tools/state-viewer/src/cli.rs

```diff
@@ -447,3 +451,22 @@ impl ApplyReceiptCmd {
         apply_receipt(home_dir, near_config, store, hash).unwrap();
     }
 }
+
+#[derive(Parser)]
+pub struct ViewTrieCmd {
+    #[clap(long)]
+    hash: String,
+    #[clap(long)]
+    shard_id: u32,
+    #[clap(long)]
+    shard_version: u32,
+    #[clap(long)]
+    max_depth: u32,
+}
```

Comment:
It would be useful to also have `prefix: Option<String>` so that I can tell `view-trie ... --max-depth=5 --prefix=1234ABCD5678` and it would start printing five levels deep from 1234ABCD7678 prefix.

Reply:
yeah - I plan to do it in a separate PR (having the prefix as you've mentioned and also an ability to pass the account_id directly)